ocaml/jane/doc/extensions/unboxed-types/index.md

```diff
@@ -284,4 +284,144 @@ Here's the list of primitives that currently support `[@layout_poly]`:
 * `%array_safe_get`
 * `%array_safe_set`
 * `%array_unsafe_get`
-* `%array_unsafe_set`
\ No newline at end of file
+* `%array_unsafe_set`
+
+# Using unboxed types in structures
+
+Unboxed types can usually be put in structures, though there are some restrictions.
+
+These structures support unboxed type fields, but have some restrictions on field
```

Comment:
I think it's a bit misleading to say these support unboxed type _fields_, since inline records don't work yet.
Maybe just 'these structures may contain unboxed types'?